repos/system_upgrade/common/actors/peseventsscanner/libraries/peseventsscanner_repomap.py

```diff
@@ -109,8 +114,8 @@ def get_pesid_repo_entry(self, repoid, major_version):
         matching_pesid_repos = []
         for pesid_repo in self.repositories:
             # FIXME(pstodulk): Why we do not check actually architecture here?
-            # It seems obvious we should check it but the fixme comment below
-            # suggests that it's expected - for not obvious reason.
+            # It seems obvious we should check it, but it's not clear why we
+            # don't and investigation might be required.
```

Comment:
AFAIK, it is not strictly necessary here since later when we are actually picking out repoids (final stages of the mapping process), we filter the candidates by architecture.